.github/agent-instructions.md

```diff
@@ -0,0 +1,168 @@
+# AI Assistant Instructions for GSM Packages
+
+**IMPORTANT:** Always read and follow the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines. This file provides GSM-specific context and patterns to supplement those guidelines.
+
+## Quick Links
+
+- **[CONTRIBUTING.md](CONTRIBUTING.md)** - **READ THIS FIRST** - Complete development workflow, branching strategy, release process
+- **Example Framework** - See [Examples section below](#examples)
+- **Package Structure** - See [Package Structure section below](#package-structure)
```

Comment:
I'd go very vanilla for this. I don't think the agent will gain anything (other than confusion) from these links.

I'd probably avoid all links throughout this. It's going to be looking at these as raw text, so including links requires an extra step. I think "the release order section of CONTRIBUTING.md" would work better than `[CONTRIBUTING.md](CONTRIBUTING.md#release-order)`, for example.